src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
Yes, the test was failed on DoCallBack_MarshalRefInt_Stdcall. Here is a gdb output for this code:
```
(gdb) n
1417	    if (sigInfo.GetCallConv() == pmCallConvCdecl)
=> 0xf6eba3e3 <UMThunkMarshInfo::RunTimeInit()+1779>:	89 04 24	mov    %eax,(%esp)
   0xf6eba3e6 <UMThunkMarshInfo::RunTimeInit()+1782>:	8b 9d e8 fe ff ff	mov    -0x118(%ebp),%ebx
   0xf6eba3ec <UMThunkMarshInfo::RunTimeInit()+1788>:	e8 ff a9 e4 ff	call   0xf6d04df0 <PInvokeStaticSigInfo::GetCallConv()>
   0xf6eba3f1 <UMThunkMarshInfo::RunTimeInit()+1793>:	3d 00 02 00 00	cmp    $0x200,%eax
   0xf6eba3f6 <UMThunkMarshInfo::RunTimeInit()+1798>:	0f 85 11 00 00 00	jne    0xf6eba40d <UMThunkMarshInfo::RunTimeInit()+1821>
(gdb) p sigInfo.GetCallConv()
$9 = pmCallConvStdcall
(gdb) p numRegistersUsed 
$10 = 1
(gdb) p m_cbActualArgSize 
$11 = 8
(gdb) p pMD->SizeOfArgStack()
$12 = 0
(gdb) p pStubMD->AsDynamicMethodDesc()->GetNativeStackArgSize()
$13 = 4

```